deployment/policies/ecs-role.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
There are ways to define these with Terraform resources. An very similar example that ties all of the IAM stuff together exists in https://github.com/azavea/raster-foundry-deployment/blob/develop/terraform/iam.tf.